src/sentry/api/serializers/models/platformexternalissue.py

```diff
@@ -7,7 +7,7 @@ class PlatformExternalIssueSerializer(Serializer):
     def serialize(self, obj, attrs, user):
         return {
             "id": str(obj.id),
-            "groupId": str(obj.group_id),
+            "issueId": str(obj.group_id),
```

Comment:
we should update https://github.com/getsentry/sentry/blob/264bc5f4d369cd9a6bed2ff18ec52b50789eeba6/src/sentry/static/sentry/app/types/index.tsx#L1260
and make sure that any frontend changes are made that need to be. I actually don't think we use the group id from this, (I think we have the actual group as a prop) but might wanna double check

Reply:
@MeredithAnya cool I updated that and poked around, the only other thing I saw was https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/actionCreators/platformExternalIssues.tsx#L9 but that's for a different endpoint anyway.